lang/rust/avro/src/schema.rs

```diff
@@ -259,56 +263,62 @@ impl Name {
     }
 
     /// Parse a `serde_json::Value` into a `Name`.
-    fn parse(complex: &Map<String, Value>) -> AvroResult<Self> {
+    pub fn parse(complex: &Map<String, Value>) -> AvroResult<Self> {
```

Comment:
could this be `pub(crate)` ?

Reply:
Ahh this is an artifact of me trying to solve the issue I made the jira issue for. Will revert